utils.py

```diff
@@ -199,7 +199,6 @@ def acquire_toolchains(self, required, registered, path, debug_calls):
             if toolchain in registered:
                 # if toolchain is registered as 'local' return empty string
```

Comment:
Thanks for the contribution. Can you also change this commit so it corresponds to what the code is doing after your change.

Reply:
You probably meant "this comment"?

Reply:
Hi, comment updated to reflect the logic change